docs/workspaces/index.md

```diff
@@ -285,7 +285,7 @@ A variety of issues can prevent a workspace from loading, including antivirus so
 - **Try another browser**: Some browser extensions may interfere with workspace loading. If you have another browser installed, try loading the workspace in that browser. Also consider trying a private/incognito window, which will disable most extensions.
 - **Check the browser developer tools console tab**: Open the browser's developer tools and navigate to the "Console" tab. This may contain error messages that can help diagnose the issue.
   - On Chrome or Edge, you can open the developer tools by opening the three-dots menu in the top right corner, selecting "More tools", and then "Developer tools".
-  - On Safari, you have to turn on the developer tools first. In the menu bar, click "Safari" -> "Settings" -> "Advanced" and check "Show features for web developers". You can then open the developer tools by selecting "Develop" -> "Show Web Inspector".
+  - On Safari, you have to turn on the developer tools first. In the menu bar, click "Safari" ⇾ "Settings" ⇾ "Advanced" and check "Show features for web developers". You can then open the developer tools by selecting "Develop" ⇾ "Show Web Inspector".
```

Comment:
I wish mkdocs did these arrows for us, I looked briefly but didn't see anything.